src/Sulu/Bundle/SnippetBundle/Content/SnippetContent.php

```diff
@@ -43,23 +43,30 @@ class SnippetContent extends ComplexContentType implements ContentTypeExportInte
      */
     private $referenceStore;
 
+    /**
+     * @var ReferenceStoreInterface
+     */
+    private $snippetAreaReferenceStore;
+
     /**
      * @var bool
      */
     protected $defaultEnabled;
 
     /**
-     * @param true $defaultEnabled
+     * @param bool $defaultEnabled
      */
     public function __construct(
         DefaultSnippetManagerInterface $defaultSnippetManager,
         SnippetResolverInterface $snippetResolver,
         ReferenceStoreInterface $referenceStore,
+        ReferenceStoreInterface $snippetAreaReferenceStore,
         $defaultEnabled
```

Comment:
This looks like a bc break think we require here some bc handling.

```suggestion
        $defaultEnabled,
        ?ReferenceStoreInterface $snippetAreaReferenceStore = null,
```